app/client/utils/roomData.ts

```diff
@@ -0,0 +1,104 @@
+import { templates } from "@reactivated";
+import { parseISO } from "date-fns/parseISO";
+
+export const ROOM_QUERY_KEY = "rooms";
+
+/** Single room data retrieved from server */
+export interface RoomAPIData {
```

Comment:
Please decide whether you use `API` (like here) or `Api` (like in React component names) - I personally recommend the latter.

Reply:
Done, I changed all of them to `Api`.